swift/swift_clang_module_aspect.bzl

```
@@ -534,12 +534,9 @@ def _handle_module(
         ),
     ]
 
-    if precompiled_module:
-        output_groups = {"swift_explicit_module": depset([precompiled_module])}
-        if indexstore_directory:
-            output_groups["indexstore"] = depset([indexstore_directory])
+    if indexstore_directory:
```

And I suppose the removal of the `precompiled_module` clause is acceptable?
```suggestion
    if precompiled_module and indexstore_directory:
```

---

Yeah, it's what upstream has. Plus it probably should have always been this way.